source/System.cpp

```diff
@@ -421,10 +421,10 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 		static const string STAR = "You cannot land on a star!";
 		static const string HOTPLANET = "This planet is too hot to land on.";
 		static const string COLDPLANET = "This planet is too cold to land on.";
-		static const string UNINHABITEDPLANET = "This planet is uninhabited.";
+		static const string UNINHABITEDPLANET = "This planet doesn't have any place suitable for landing.";
```

Comment:
Perhaps just remove the question of suitability and say there's no place you can land?
I prefer `anywhere` to `any place`, but that may be subjective.
```suggestion
		static const string UNINHABITEDPLANET = "This planet doesn't have anywhere you can land.";
```
```suggestion
		static const string UNINHABITEDPLANET = "This planet doesn't have any place you can land.";
```
Alternatively:
```suggestion
		static const string UNINHABITEDPLANET = "You can't land on this planet.";
```

Reply:
I think the first suggestion is clearer than both the PR and the second one; the second one doesn't give a reason and it looks like an arbitrary restriction.